X/X265Builder/build_tarballs.jl

```diff
@@ -0,0 +1,45 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "x265Builder"
+version = v"3.0"
+
+# Collection of sources required to build x265Builder
+sources = [
+    "http://ftp.videolan.org/pub/videolan/x265/x265_3.0.tar.gz" =>
+    "c5b9fc260cabbc4a81561a448f4ce9cad7218272b4011feabc3a6b751b2f0662",
+
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd x265_3.0/
+apk add nasm
+export PKG_CONFIG_PATH="${prefix}/lib/pkgconfig"
+mkdir bld && cd bld
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=/opt/$target/$target.toolchain -DENABLE_PIC=ON -DENABLE_SHARED=off ../source
```

Comment:
Now there is an environment variable that points to the location of the cmake toolchain file (cannot easily check its name on my phone)

Reply:
```suggestion
cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE="${CMAKE_TARGET_TOOLCHAIN}" -DENABLE_PIC=ON -DENABLE_SHARED=off ../source
```